packages/backend/src/auth/auth.service.ts

```diff
@@ -62,4 +68,89 @@ export class AuthService {
   async validateUser(payload: SignInPayload) {
     return await this.usersService.findByPayload(payload);
   }
-}
+
+  async resetPassword(email: string) {
+    try {
+
+      const user = await this.usersService.findByEmail(email);
```

Comment:
to sie powtarza z tym errorem w paru metodach moze to tez do osobnej metody?

Reply:
poprawiłem